code/datums/uplink/devices and tools.dm

```diff
@@ -71,9 +71,9 @@
 	path = /obj/item/weapon/storage/box/syndie_kit/space
 
 /datum/uplink_item/item/tools/thermal
-	name = "Thermal Imaging Glasses"
+	name = "Thermal Imaging Lenses"
 	item_cost = 6
```

Comment:
It needs to be 10 for the lack of cells needed.

For this reason as well, we can have both of those left in uplink.

Reply:
#75bfb48